modules/preview-react/index.ts

```diff
@@ -7,5 +7,3 @@ export * from './select';
 export * from './side-panel';
 export * from './text-area';
 export * from './text-input';
-
-/// <reference types="@types/node" />
```

Comment:
Not sure why this was here, but it was breaking being able to create a preview component. I didn't see anything like this in other index files, so removing seems safe?

Reply:
Yeah, no idea why that's there.

Reply:
Okay, I read up on [triple-slash directives](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html). And based on this, I think it's safe to remove it.

> Triple-slash directives are only valid at the top of their containing file. A triple-slash directive can only be preceded by single or multi-line comments, including other triple-slash directives. If they are encountered following a statement or a declaration they are treated as regular single-line comments, and hold no special meaning.

That said, if all our types blow up, this will be the line we need to add back.

Reply:
yoloooooo

Reply:
I checked the blame and @NicholasBoll added an explanation for why this line exists: https://github.com/Workday/canvas-kit/pull/1408/files#r779218020

Reply:
interesting, thanks for looking that up.  I wasn't getting any type errors running storybook, but could have missed something.

Reply:
If there are no type errors, I'm glad to have it removed.